buildpack/bomfile.go

```diff
@@ -13,6 +15,13 @@ const (
 	LayerTypeLaunch
 )
 
+const (
+	mediaTypeCycloneDX   = "application/vnd.cyclonedx+json"
+	mediaTypeSPDX        = "text/spdx+json"
```

Comment:
This ended up being registered as `application/spdx+json` per https://github.com/spdx/spdx-spec/issues/577#issuecomment-960295523

Reply:
Thanks. _Fixed in latest commit._